recipe/meta.yaml

```
@@ -55,28 +46,28 @@ outputs:
     script: install_llvm.bat  # [win]
     build:
       activate_in_script: true
```

```suggestion
      activate_in_script: true
      run_exports:
        - {{ pin_subpackage("libllvm" + major_ver, max_pin="x.x") }}
```
Assuming that `llvmdev` provides the API for `libllvmX`.